### PR TITLE
ruby dependencies for gentoo

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
https://github.com/ros/rosdistro/issues/15082 for reference

File: rosdep/ruby.yaml [hunk 1]
```diff
@@ -77,7 +85,9 @@ ruby:
 ruby-sass:
   debian: [ruby-sass]
   fedora: [rubygem-sass]
+  gentoo: [dev-ruby/sass]
   ubuntu: [ruby-sass]
 ruby1.9.3:
+  gentoo: []
```
Comment (mikaelarguedas) [inline rosdep/ruby.yaml]: do you know of any package using this ? I think this can be removes given that no released package on the buildfarm uses it

**Review by mikaelarguedas — APPROVED** [review]
lgtm, except one comment about adding rules to unused keys

File: rosdep/ruby.yaml (hunk 1 above)
Comment (Alessandro-Barbieri) [inline rosdep/ruby.yaml]: It's upstream deprecated and no more available on gentoo. I don't know any package that uses it.

Comment (mikaelarguedas) [inline rosdep/ruby.yaml]: feel free to remove this line then

**Comment by mikaelarguedas** [comment]
LGTM

**Pull request merged**